app/modules/key_management/secret_manager.py

```
@@ -1408,3 +1445,269 @@
             "successful_deletions": successful_deletions,
             "not_found": not_found,
         }
+
+    # Workflow secret endpoints
+
+    @router.post("/workflow-secrets")
+    async def create_workflow_secrets(
+        request: CreateWorkflowSecretRequest,
+        user=Depends(AuthService.check_auth),
```

Ruff (0.11.9)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)